core/embed/extmod/modtrezorcrypto/modtrezorcrypto-secp256k1_zkp.h

```diff
@@ -291,24 +374,31 @@ STATIC MP_DEFINE_CONST_FUN_OBJ_3(
 
 static int secp256k1_ecdh_hash_passthrough(uint8_t *output, const uint8_t *x,
                                            const uint8_t *y, void *data) {
-  output[0] = 0x04;
+  output[0] = SECP256K1_TAG_PUBKEY_UNCOMPRESSED;
   memcpy(&output[1], x, 32);
   memcpy(&output[33], y, 32);
   (void)data;
   return 1;
 }
 
-/// def multiply(self, secret_key: bytes, public_key: bytes) -> bytes:
+/// def multiply(
+///     self, secret_key: bytes, public_key: bytes, compressed_result: bool =
+///     False
```

Comment:
nit: linebreak

Reply:
I have reverted the change to `multiply` - the compression can be done in Python.